### DISCUSSION TITLE
How to log in to stellarx

### DISCUSSION THREAD

**Discussion by gityms123** [WalletConnect/walletconnect-monorepo#1101] [Developer Support]
I found on the official website [](https://walletconnect.com/registry?type=dapp&version=2) that stellarx supports V2, but the QR code for stellarx login is like this,**wc:5f1cb4c0a60ad299993344903d33feaa9bc2b3f80900f826f5a4fd01137fb66c@2?controller=false&publicKey=3b250075ba24f1a0131fdcb75d88d29dabe009b3f75ca9a03227f786b1a22135&relay=%7B%22protocol%22%3A%22waku%22%7D**
Even the validateWCUri method fails to pass

**Comment by chadyj** [comment]
StellarX is indeed using v2, so you will need a v2 compatible wallet.

You might notice that Stellarx recommends Lobstr Wallet on their connect UI.  Give that a try.

<img width="594" alt="Screen Shot 2022-06-18 at 9 05 03 AM" src="https://user-images.githubusercontent.com/116099/174426920-69c79ef1-54fc-42ef-98f0-c9217d3aeb2d.png">
